common/trtexec_vendor/CMakeLists.txt

```
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.14)
+project(trtexec_vendor)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(CUDA REQUIRED)
+find_package(cudnn_cmake_module REQUIRED)
+find_package(CUDNN REQUIRED)
+find_package(tensorrt_cmake_module REQUIRED)
+find_package(TENSORRT REQUIRED)
```

```suggestion
find_package(ament_cmake)
find_package(CUDA)
find_package(cudnn_cmake_module)
find_package(CUDNN)
find_package(tensorrt_cmake_module)
find_package(TENSORRT)
```
To prevent from build error for non-gpu environment.

---

@yukke42 I guess we should keep `ament_cmake`, `cudnn_cmake_module`, `tensorrt_cmake_module` required because they are just cmake helpers. What dou you think?

---

Sorry, It's just a mistake. I will update.